providers/celery/src/airflow/providers/celery/executors/celery_kubernetes_executor.py

```diff
@@ -67,7 +67,6 @@ class CeleryKubernetesExecutor(BaseExecutor):
     is_production: bool = True
 
     serve_logs: bool = False
-    change_sensor_mode_to_reschedule: bool = False
```

Comment:
Oh, we should leave these in the providers so that they still work with Airflow 2.x

Reply:
Oh, but it's the default form base executor, so it should be safe to remove actually.